docs/content/en/manifest.md

```diff
@@ -32,7 +32,7 @@ pwa: {
 | `start_url` <sup>\*1</sup>        | `String`        | `routerBase + '?standalone=true'`                            | It has to be relative to where the manifest is placed           |
 | `display` <sup>\*1</sup>          | `String`        | `'standalone'`                                               |                                                                 |
 | `background_color` <sup>\*2</sup> | `String`        | `'#ffffff'`                                                  |                                                                 |
-| `theme_color` <sup>\*2</sup>      | `String`        | `this.options.loading.color`                                 | Nuxt [loading color] option                                     |
+| `theme_color` <sup>\*2</sup>      | `String`        | `this.options.pwa.meta.theme_color`                          | This module's meta theme-color (see the [meta module])          |
```

Comment:
I think mentioning `this.options.*` is not necessary as `theme_color` can be set same as rest in table. We only removed default

Reply:
>  If the docs say it's undefined but then has a value, even when not set, it might be confusing. What do you think?

Yes this completely makes sense. We could just simplify content (see 46694263a4dec45261017cd40871e4492fec8753)